assets/dev/js/editor/document/hooks/ui/settings/reload-preview.js

```diff
@@ -0,0 +1,35 @@
+import After from 'elementor-api/modules/hooks/ui/after';
+
+export class ReloadPreview extends After {
+	getCommand() {
+		return 'document/elements/settings';
+	}
+
+	getId() {
+		return 'save-layout';
+	}
+
+	getContainerType() {
+		return 'document';
+	}
+
+	getConditions( args ) {
+		return !! args.settings.template;
+	}
+
+	apply() {
+		// Logic moved from js assets/dev/js/editor/components/settings/page/manager.js
```

Comment:
Use this comment as part of the commit, not as an inline comment

Reply:
Why?

Reply:
@Nevoss move to the commit description.